product-updates/knowledge-center/index.html

```diff
@@ -335,8 +335,77 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
 
         <h2>What's new in TDP</h2>
 
-        <h3 style="margin-top: 32px;">December 18th 2024 (v 3.7.5)</h3>
+        <h3 style="margin-top: 32px;">February 11th, 2025 (v 3.7.7)</h3>
 
+        <h4 style="margin-top: 16px; color: #595959;">Changed / Fixed:</h4>
+        <ul class="usa-icon-list">
+          <li class="usa-icon-list__item" style="margin-top: 16px;">
+            <div class="usa-icon-list__icon text-ink">
+                <img src="../img/changed.svg" aria-hidden="true" alt="" style="background-color: #b3e2ff; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+          </div>
+          
+          <div class="usa-icon-list__content">
+            <h5 class="usa-icon-list__title">
+              Refinements to the Critical Tab in Error Reports
+            </h5>
+            <p>We've updated the Critical Tab within the multi-tab error report to provide better visibility into the most important errors to review first. The Critical Tab now includes errors related to information required to identify families receiving assistance and subsets of work-eligible families. This includes, but is not limited to, errors related to Family Affiliation and Benefit Amount. These changes ensure that errors impacting our ability to calculate key metrics, such as the Work Participation Rate, are surfaced more prominently.</p>
```

Comment:
```suggestion
            <p>We've updated the Critical Tab within the multi-tab error report to provide better visibility into the most important  Section 3: Aggregate and Section 4: Stratum file errors to review first. </p>
```

Excluding the rest because this was always the case, but we recently elevated checks for sec3 and sec4 (#3381) @klinkoberstar